### ISSUE TITLE
Broken copy & paste in Mac Sierra with DBeaver 4.21

### ISSUE THREAD

**Issue by alanwilter** [serge-rider/dbeaver#2168]
On my mac, cmd+C and cmd+V do not work. I can use mouse right button but if I copy from another application to DBeaver, I got this message:

Clipboard contains data in unsupported format

Please, this is quite serious, at least for me, and renders Dbeaver unusable.

**Comment by alanwilter** [comment]
Moreover, any cmd+KEY, like cmd+Z, is not working anymore. I can report this was all fine in 4.1.

**Comment by serge-rider** [comment]
Please provide a bit more details: where are you trying to perform copy/paste? 
Data editor, database navigator, some dialog?

**Comment by alanwilter** [comment]
Ok, I've tested again at home and all seems fine. Let me try again at work, and then if all fine, I may even close this issue.

**Comment by alanwilter** [comment]
Indeed, I can't reproduce this anymore so I don't know what was the cause. I am closing it.
